Source/MLX/MLXArray.swift

```diff
@@ -246,6 +241,109 @@ public final class MLXArray {
         asType(T.dtype, stream: stream)
     }
 
+    /// Return the dimension where the storage is contiguous.
+    ///
+    /// If this returns 0 then the whole storage is contiguous.  If it returns ndmin + 1 then none of it is contiguous.
+    func contiguousToDimension() -> Int {
```

Comment:
This is a private method?

Reply:
Yeah, it is private to the library but callable from tests.